packages/govuk-frontend/src/govuk/components/checkboxes/_index.scss

```diff
@@ -289,9 +289,8 @@
         outline-color: Highlight;
       }
       box-shadow:
-        0 0 0 $govuk-focus-width $govuk-focus-colour,
-        // 1
-        0 0 0 $govuk-hover-width $govuk-hover-colour; // 2
+        0 0 0 $govuk-focus-width $govuk-focus-colour /* 1 */,
+        0 0 0 $govuk-hover-width $govuk-hover-colour; /* 2 */
```

Comment:
**nitpick** This will output the `/* 1 */` in the final CSS, but the `[1]` it refers to in the comment above the selector will be stripped out, being just a Sass comment. I think it's fine to `prettier-ignore` here to keep the formatting.

```suggestion
      // prettier-ignore
      box-shadow:
        0 0 0 $govuk-focus-width $govuk-focus-colour, // 1
        0 0 0 $govuk-hover-width $govuk-hover-colour; // 2
```

Reply:
Interesting formatting from Github that picked both the commit's deletion and addition... 🤔

Reply:
✅ Done